packages/backend/apps/multitenancy/services/membership.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib.auth import get_user_model
+
+from ..models import Tenant, TenantMembership
+from ..constants import TenantUserRole
+
+User = get_user_model()
+
+
+def create_tenant_membership(user: User, tenant: Tenant, role: TenantUserRole = TenantUserRole.OWNER):
```

Comment:
I don't think "owner" is a good choice for a default membership type. Default permissions should be as restrictive as possible IMO.